LDRH_choice.py

```diff
@@ -460,16 +491,28 @@ def run_staircase(task, operation=None):
     except StopIteration:
         output={}
         output['score'] = 'StopIteration'
-        output['thisIncrement'] = handler.intensities[-1]
+        if not use_posterior_matching:
+            output['thisIncrement'] = handler.intensities[-1]
 
     return output
 
+if testing_questionnaire:
```

Comment:
Why does the Questionnaire codeblock appear twice? Shouldn't it only appear at the end?